docs/data/base/components/slider/slider.md

```diff
@@ -0,0 +1,60 @@
+---
+product: base
+title: React Slider unstyled component and hook
+components: SliderUnstyled
+githubLabel: 'component: slider'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#slider
+packageName: '@mui/base'
+---
+
+# Slider
+
+<p class="description">Sliders allow users to make selections from a range of values.</p>
+
+Sliders show a range of values along a bar, from which users may select a single value.
+They are ideal for adjusting settings such as volume, brightness, or applying image filters.
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+```js
+import SliderUnstyled from '@mui/base/SliderUnstyled';
+```
+
+{{"demo": "UnstyledSlider.js", "defaultCodeOpen": false}}
+
+## Discrete sliders
+
+Discrete sliders snap to the nearest step.
+You can generate a mark for each step with `marks={true}`.
```

Comment:
Let's make this more informative. Also, I think "stop" is a better term than "step" (which implies vertical movement), following [the usage in this article](https://babich.biz/mobile-ux-design-sliders/).

```suggestion
The most basic slider is _continuous_, which means it does not have pre-defined (_discrete_) values for the user to select from.
This is suitable for situations in which an approximate value is good enough for the user, such as brightness or volume.

But if your users need more precise options, you can create a discrete slider with pre-defined stops along the bar.
To generate a mark for each stop, use `marks={true}`:
```